integration-tests/athenaQuery.spec.ts

```diff
@@ -0,0 +1,89 @@
+import {
+  populateDynamoDBWithRequestDetails,
+  getValueFromDynamoDB
+} from './utils/aws/dynamoDB'
+import { addMessageToQueue } from './utils/aws/sqs'
+import {
+  getCloudWatchLogEventsGroupByMessagePattern,
+  isLogPresent
+} from './utils/aws/cloudWatchGetLogs'
+import { createZendeskTicket } from './utils/zendesk/createZendeskTicket'
+import { validRequestData } from './constants/requestData'
+import { approveZendeskTicket } from './utils/zendesk/approveZendeskTicket'
+import {
+  INITIATE_ATHENA_QUERY_LAMBDA_LOG_GROUP,
+  INITIATE_ATHENA_QUERY_QUEUE_URL
+} from './constants/awsParameters'
+import { deleteZendeskTicket } from './utils/zendesk/deleteZendeskTicket'
+
+describe('Athena Query SQL generation and execution', () => {
+  jest.setTimeout(60000)
+  let ticketId: string
+
+  beforeEach(async () => {
```

Comment:
If we're just putting messages directly in the queue. Do we need to create a Zendesk ticket? This will cause 2 conflicting processes to run surely?